src/video_emotion/extract_test_data.py

```diff
@@ -1,27 +1,92 @@
 import sys
+import os
+from os import listdir
+from os.path import isfile, join
 
-from video_emotion.extract_classifier import classify_video
+sys.path.insert(0, os.path.abspath("../"))
+
+import cv2
+import numpy as np
 
+from video_emotion.extract_classifier import classify_video
+from video_emotion.facial_recognition.facial_recognition import analyze_video
+from video_emotion.emotionTagger.face_emotion_extraction import classify_faces
 
 def generate_log_data(interval, emotion_set):
     output_log_data = ""
```

Comment:
Why is this line needed, instead of just the one below, without the '+'

Reply:
sure

Reply:
Resolved